assets/javascripts/test_result.js

```diff
@@ -324,6 +326,24 @@ function setupTab(tabHash) {
     }
 }
 
+function setupInvestigation() {
+    var element = document.getElementById('investigation_status');
+    $.ajax({
+        url: element.dataset.url,
+        method: 'GET',
+        success: function(response) {
+            document.innerHTML = '';
+            var div = document.createElement('pre');
```

Comment:
Maybe this is better called `preElement` now.

Reply:
done